src/harness/unittests/tscWatchMode.ts

```diff
@@ -1016,46 +1016,46 @@ namespace ts.tscWatch {
             checkProgramActualFiles(watch(), [f.path, libFile.path]);
         });
 
-        it("Options Diagnostic locations reported correctly with changes in configFile contents when options change", () => {
-            const file = {
-                path: "/a/b/app.ts",
-                content: "let x = 10"
-            };
-            const configFileContentBeforeComment = `{`;
-            const configFileContentComment = `
-                    // comment
-                    // More comment`;
-            const configFileContentAfterComment = `
-                    "compilerOptions": {
-                        "allowJs": true,

```

Comment:
you could change this to set { "inlineSourceMap": true, "sourceMap": true } to make sure the errors are reported on those lines in the config file.